src/Esfa.Vacancy.Register.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Esfa.Vacancy.Api.Types;
+using Esfa.Vacancy.Register.Api.Attributes;
+using Esfa.Vacancy.Register.Api.Orchestrators;
+using Swashbuckle.Swagger.Annotations;
+
+namespace Esfa.Vacancy.Register.Api.Controllers
+{
+    [ExceptionHandling]
```

Comment:
Is this not set as Global ActionFilter? Not sure you need to declare yourself on Controllers.